tools/developer/run.sh

```diff
@@ -186,7 +185,7 @@ function test_compile {
     echo --------------------------------------------
     for d in ${TAP_DIRS}
     do
-        time bash taptest.sh  "${d}" "-p ${PGPORT}"
+        time bash build/taptest.sh  "${d}" "-p ${PGPORT}"
```

Comment:
Looks like you moved taptest.sh to build
The copy of taptest.sh is to the root of the repo.

Undo this modification